Solutions/KasperskySecurityCenter/Data Connectors/Connector_KasperskySC_CEF.json

```diff
@@ -33,7 +33,7 @@
     ],
     "availability": {
         "status": 1,
-        "isPreview": true
+        "isPreview": false
     },
     "permissions": {
         "resourceProvider": [

```

Comment:
Hi @vmanojreddy, please change data connector text from `Azure Sentinel` to `Microsoft Sentinel`.